package.json

```diff
@@ -157,9 +157,40 @@
     "packages/*"
   ],
   "jest": {
+    "collectCoverage": true,
+    "coverageProvider": "v8",
+    "testPathIgnorePatterns": [
+      "<rootDir>/packages/spec/.tmp/"
+    ],
+    "watchPlugins": [
+      "jest-runner-eslint/watch-fix",
+      "jest-watch-select-projects",
+      "jest-watch-typeahead/filename",
+      "jest-watch-typeahead/testname"
+    ],
     "projects": [
       "<rootDir>/packages/!(spec)",
-      "<rootDir>/packages/spec/integration/*"
+      "<rootDir>/packages/spec/integration/*",
+      {
+        "displayName": "lint",
+        "runner": "eslint",
+        "testPathIgnorePatterns": [
+          "<rootDir>/coverage/"
+        ],
+        "testMatch": [
+          "<rootDir>/packages/**/*.js"
+        ]
+      },
+      {
+        "displayName": "prettier",
+        "runner": "prettier",
+        "preset": "jest-runner-prettier",
+        "testPathIgnorePatterns": [
```

Comment:
Since this runner apparently [does not respect `.prettierignore`](https://github.com/keplersj/jest-runner-prettier/issues/87), we should add `package.json` and `package-lock.json` here.

Reply:
We don't use npm, so we don't have a `package-lock.json`, do you think we should still add it?
And I think having the `package.json` formatted by prettier is ok nowadays. We excluded it initially because a `yarn add` or `npm install` would result in a differently formatted file, but as far as I can see this is no longer an issue, right?

Reply:
Hm, haven't checked that in a while. I'm happy to not have `package.json` ignored by Prettier. But I think we should still ignore `package-lock.json`, because npm is the package manager referenced in the docs and contributors might use it.